packages/react-auth/src/AuthSignIn.tsx

```diff
@@ -143,7 +145,7 @@ export const AuthSignIn = ({
           </Text>
         </Flex>
 
-        <NotificationsBox />
+        <AuthNotifications />
```

Comment:
I believe that it should be kept as `NotificationsBox` and remove the component `AuthNotifications` of react-auth package